src/routes/Root/Root.tsx

```diff
@@ -11,26 +10,12 @@ import './Root.css';
 
 const Root = () => {
   const { t, i18n } = useTranslation();
-
-  const links = [
-    { to: `/${i18n.language}`, text: t('home') },
-    { to: `/${i18n.language}/${t('slugs.profile.index')}`, text: t('profile') },
-    { to: `/${i18n.language}/${t('slugs.about-us')}`, text: t('about-us') },
-    { to: `/${i18n.language}/${t('slugs.about-us')}/${t('slugs.terms-of-service')}`, text: t('terms-of-service') },
-    {
-      to: `/${i18n.language}/${t('slugs.about-us')}/${t('slugs.accessibility-statement')}`,
-      text: t('accessibility-statement'),
-    },
-    { to: `/${i18n.language}/${t('slugs.about-us')}/${t('slugs.privacy-policy')}`, text: t('privacy-policy') },
-    { to: `/${i18n.language}/${t('slugs.about-us')}/${t('slugs.cookies')}`, text: t('cookies') },
-  ];
```

Comment:
Did remove the links for now, but did not yet remove the pages.
Some of these would be available from the footer that is later down when design updates regarding that.